scripts/us_eia/opendata/process/test_data/COAL.csv

```diff
@@ -0,0 +1,2879 @@
+place,stat_var,date,value,unit,scaling_factor,eia_series_id

```

Comment:
I wonder if we should curate the input COAL.txt a bit, to make it more readable/editable?

I have been trimming series to 2-3 data points, and picked 2 or 3 series that span patterns.

Reply:
oh good point. i selected a 1% random sample of the file. it's helped to catch some other errors while i was adding patterns. i'll trim down some of the datapoints, and reduce the number of series.

Reply:
Thanks!